docs/en/modules/cron.md

```diff
@@ -110,9 +104,9 @@ ent:schedule("*/5 * * * *")
 
 ## cron.entry:unschedule()
 
-Disable schedule.
+Disables schedule.
 
-Disabled schedule may be enabled again by calling [`:schedule(mask)`](#cronentryschedule).
+Disabled schedules may be enabled again by calling [`:schedule(mask)`](cron.md#cronentryschedule).
```

Comment:
Not for viewing files on GitHub but for MkDocs (which converts to HTML on readthedocs.io) if I'm not mistaken.